pkg/agent/main.go

```diff
@@ -40,6 +42,17 @@ var (
 func main() {
 	agentSpec := controller.AgentSpecification{}
 
+	// Handle environment variables:
+	// LIGHTHOUSE_VERBOSITY determines the verbosity level
+	// LIGHTHOUSE_DEBUG, if set to true, sets the verbosity level to 3
+	if debug := os.Getenv("LIGHTHOUSE_DEBUG"); debug != "" {
+		os.Args = append(os.Args, "-v=3")
+	} else if verbosity := os.Getenv("LIGHTHOUSE_VERBOSITY"); verbosity != "" {
```

Comment:
Can we just go with "SUBMARINER_" like with metrics? I think we have a SUBMARINER_DEBUG flag in our CRD, so we'll be able to reuse that, right?

Reply:
Done. CRD handling has to be done in the operator, but at least if the variables are the same everywhere, it will be easier to understand!